README.md

```diff
@@ -32,6 +32,7 @@ configured, take a look at the EarlGrey API and start writing your own tests.
   * [Backward Compatibility](https://github.com/google/EarlGrey/tree/master/docs/backward-compatibility.md)
   * [Install and Run](https://github.com/google/EarlGrey/tree/master/docs/install-and-run.md)
   * [API](https://github.com/google/EarlGrey/tree/master/docs/api.md)
+  * [Cheat Sheet](docs/cheatsheet/cheatsheet.png)
```

Comment:
There are a bunch but they all look kind of spammy. I think committing a rendered png is a better user experience.